doc/user/content/sql/alter-cluster.md

```diff
@@ -15,8 +15,21 @@ cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
 
 {{< diagram "alter-cluster-reset.svg" >}}
 
+### `with_options`
+
+{{< private-preview />}}
+
+{{< diagram "with-options-alter-cluster-set.svg" >}}
+
```

Comment:
Should we add a section heading before the next line? because the `{cluster-options}` are for the top level Alter Clusters option, instead of `### with_options` (on line 18).  Either that, or maybe the  `{cluster-options}`  needs to move up?

Reply:
I think I tried to model this after https://materialize.com/docs/sql/create-source/postgres/#with_options
and a few others. Not to say that they're correct.

It is also possible to add the `with` section to the `alter cluster` diagram.

Reply:
I'll patch this, too.